### ISSUE TITLE
Can't change service in Android app?

### ISSUE THREAD

**Issue by phlummox** [xBrowserSync/App#10]
Not sure if this is just me, or not, but: I successfully created my own xBrowserSync server instance, and was able to use it from the browser, and via the Google Chrome extension. However, if I attempted to change the server being used from the Android app, no error message was given, but nothing happened. The server instance was running via HTTPS, and had a valid TLS certificate, but was running on a non-standard port.

(As an aside: I noticed that the Android app wouldn't let me insert an unqualified hostname in the URL for the server (e.g. "https://myhost:8080/"). I can see there are reasons for avoiding that, but it certainly can be handy for testing purposes - maybe it could be enabled, but with a warning?)

### Steps to reproduce
1. Under "Settings" in the Android app, select "Change service"
2. Enter a URL in the URL box - e.g. "https://myhost.net:9000/"
3. Click "update" button

### Expected behaviour
App should use new URL for the server.

### Actual behaviour
App pauses for a fraction, but nothing changes.

### Client app information
Android

**App Platform Version:** 
6.0

**Comment by nero120** [comment]
Hi @phlummox, could you just paste the response you get when you browse to `[your xBrowserSync API host URL]/info` so I can verify the config?

**Comment by phlummox** [comment]
Hi, no worries - will do so when I get a chance. Have just started my first day at a new job though, so I may not get to it til later this week, tho.. cheers!

**Comment by Shadowpheonix** [comment]
Not entirely sure if it is relevant (especially given the age of this defect), but I also was unable to change the service in the Android app when I tried to use my site with a self-signed SSL certificate.  However, I was able to change the service successfully when I just used HTTP instead.  I am fairly certain the issue was with my SSL certificate.
When trying to go to the /info page in Chrome, I get a "your connection is not private" warning, but I can use the Advanced option and continue successfully, getting the expected JSON server status info.

**Comment by nero120** [comment]
Yeah it's because it's self signed. Once you have told Chrome to accept the self signed cert, can you connect ok?

Also, not sure if you're aware but since we now have the [Let's Encrypt](https://letsencrypt.org/) cert authority there's no need to use self signed certs as you can generate your own valid SSL certs for free. If you don't want to set up your own Let's Encrypt service then you can use a site like [ZeroSSL](https://zerossl.com/) to get your SSL cert, just bear in mind that it's better to generate your own private keys than via a third party.

**Comment by Shadowpheonix** [comment]
@nero120 Thank you for that info.  I had actually never heard of either service and the only info I could find on free SSL certs via my Google searching (two years ago) was for the self-signed method.
I used ZeroSSL to get a cert for my server, and I now have everything working perfectly with HTTPS.

This is the second time you have helped me with an issue that turned out to be on my side (the first was via email), and I _**REALLY**_ appreciate it.
